lib/gbhw.fs

```diff
@@ -10,6 +10,7 @@ also gb-assembler
 
 : [rLY]   $FF44 ]* ;
 : [rSTAT] $FF41 ]* ;
+[endhost]
 
 %00000000 constant LCDCF_OFF        ( LCD Control Operation)
```

Comment:
Would maybe make sense to keep `constant` in the host as well?

Reply:
Well. I think constants are always host right?  Bit if they are static there is no need to switch to host.  It's kind of declarative thing.   We should minimize host context though. It is a hack for missing things in the vocabulary for target. 

It's a good idea to add words to allow to do that declaratively